Frameworks/CoreGraphics/CGColorSpace.mm

```diff
@@ -175,15 +175,16 @@ size_t CGColorSpaceGetNumberOfComponents(CGColorSpaceRef pSpace) {
  @Status Interoperable
 */
 void CGColorSpaceRelease(CGColorSpaceRef colorSpace) {
-    CFRelease((id)colorSpace);
+    RETURN_IF(!colorSpace);
+    CFRelease(static_cast<CFTypeRef>(colorSpace));
```

Comment:
wait, don't reset that. both of these are More Correct :tm: than what was there.